build/source/dshare/popMetadat.f90

```diff
@@ -819,100 +846,105 @@ subroutine popStat(meta,lineWords,nWords,indexFlags,err,message)
  ! initiate error handling
  err=0; message='popStat/'
 
+ ! get the number of words in a line
+ nWords = size(lineWords)
+
  ! make sure the variable only has one output frequency
- found = .false.
- do iFreq = 1,nFreq
-  do iStat = 1,maxVarStat
-   if (meta%statFlag(iStat)) then
-    if (found) then
-     err=20
-     message=trim(message)//'cannot have more than one file line per variable:'//trim(meta%varName)
-     return
-    end if
-    found = .true.
-    exit
-   end if
-  end do
- end do
+ if(count(meta%statFlag)>0)then
+  message=trim(message)//'model output for variable '//trim(meta%varName)//' already defined (can only be defined once)'
+  err=20; return
+ endif
```

Comment:
@luckymichael This was simply cleaning up code from @greyNearing. We currently have the restriction that there is a single timescale for each variable, with the understanding that the use can always temporally aggregate using `nco` commands.

I'm not sure about the reason for this restriction, likely to simplify the metadata structures. @greyNearing wrote the original code and he could comment.